src/databases/Mili/avtMiliFileFormat.C

```diff
@@ -2755,13 +2759,14 @@ avtMiliFileFormat::PopulateDatabaseMetaData(avtDatabaseMetaData *md,
                 for (int j = 0; j < groupShared.size(); ++j)
                 {
                     bool addVarNow = true;
+                    SharedVariableInfo *sharedInfo = NULL;
```

Comment:
It seems like there isn't any reason do define this variable one level out in the scope. Can you add the declaration back to line 2769.

Reply:
I moved it out of scope so that it can be accessed at line 2804.